edge-modules/TwinTester/TwinEdgeOperationsResultHandler.cs

```diff
@@ -4,36 +4,40 @@ namespace TwinTester
     using System;
     using System.Threading.Tasks;
     using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResultCoordinatorClient;
+    using Microsoft.Azure.Devices.Edge.Util;
     using Microsoft.Azure.Devices.Shared;
     using Microsoft.Extensions.Logging;
-    using Newtonsoft.Json;
-    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResultCoordinatorClient;
 
     class TwinEdgeOperationsResultHandler : ITwinTestResultHandler
     {
         static readonly ILogger Logger = ModuleUtil.CreateLogger(nameof(TwinEdgeOperationsResultHandler));
         readonly TestResultCoordinatorClient trcClient;
         readonly string moduleId;
+        readonly Option<string> trackingId;
 
-        public TwinEdgeOperationsResultHandler(Uri analyzerClientUri, string moduleId)
+        public TwinEdgeOperationsResultHandler(Uri analyzerClientUri, string moduleId, Option<string> trackingId)
```

Comment:
```suggestion
        public TwinEdgeOperationsResultHandler(Uri reportUri, string moduleId, Option<string> trackingId)
```

Reply:
is this class only used in Connectivity test?

Reply:
yes, it's used for connectivity only